scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -522,16 +550,19 @@ function Install-IoTEdge {
         # IoT Edge Agent image to pull for the initial configuration.
         [Parameter(ParameterSetName = 'Manual')]
```

Comment:
Should just remove the `Parameter` attributes here entirely, just like in `Initialize-IoTEdge`

Same for `$Username` and `$Password`